molecule/user_provided/vars/vars.yml

```diff
@@ -0,0 +1,25 @@
+tas_single_node_oidc_issuers:
+  - issuer: "https://keycloak-keycloak-system.apps.platform-sts.pcbk.p1.openshiftapps.com/auth/realms/trusted-artifact-signer"
+    url: "https://keycloak-keycloak-system.apps.platform-sts.pcbk.p1.openshiftapps.com/auth/realms/trusted-artifact-signer"
+    client_id: trusted-artifact-signer
+    type: email
+tas_single_node_base_hostname: myrhtas
+tas_single_node_cockpit:
+  enabled: false
+tas_single_node_registry_username: "{{ lookup('env', 'TAS_SINGLE_NODE_REGISTRY_USERNAME') }}"
+tas_single_node_registry_password: "{{ lookup('env', 'TAS_SINGLE_NODE_REGISTRY_PASSWORD') }}"
+tas_single_node_trillian:
+  database_deploy: false
+  mysql:
+    user: mysql
+    root_password: rootpassword
+    password: password
+    database: trillian
+    host: localhost
```

Comment:
Does localhost work here? I don't understand how it would, because that would make the trillian container try to reach itself rather than the host that it's running on. I think this should be the molecule host IP, right (same for Redis)?